homeassistant/components/ferienapidotde/binary_sensor.py

```diff
@@ -0,0 +1,149 @@
+"""Binary sensor to indicate if today is a german vacation day or not.
+
+Utilizes the api `ferien-api.de` to provide a binary sensor to indicate if
+today is a german vacation day or not - based on your configured state.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.ferienapidotde/
+"""
+
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.util import Throttle
```

Comment:
Please keep the import sorted. `isort` can help if you don't want to do it manually.